### PR TITLE
Prebid SDK Rendering API Extensions

### PR DESCRIPTION
**Pull request by SyntaxNode**
For our Prebid SDK friends, per https://github.com/prebid/prebid-server/issues/2908.

### CODE REVIEW

File: exchange/utils.go [hunk 1]
```diff
@@ -363,12 +363,13 @@ func buildRequestExtForBidder(bidder string, requestExt json.RawMessage, request
...
-		prebid.CurrencyConversions = requestExtParsed.Prebid.CurrencyConversions
-		prebid.Integration = requestExtParsed.Prebid.Integration
 		prebid.Channel = requestExtParsed.Prebid.Channel
+		prebid.CurrencyConversions = requestExtParsed.Prebid.CurrencyConversions
 		prebid.Debug = requestExtParsed.Prebid.Debug
-		prebid.Server = requestExtParsed.Prebid.Server
+		prebid.Integration = requestExtParsed.Prebid.Integration
 		prebid.MultiBid = buildRequestExtMultiBid(bidder, requestExtParsed.Prebid.MultiBid, alternateBidderCodes)
+		prebid.Sdk = requestExtParsed.Prebid.Sdk
+		prebid.Server = requestExtParsed.Prebid.Server
```
Comment (SyntaxNode) [inline exchange/utils.go:372]: Alphabetized the list and added `sdk`.

File: openrtb_ext/bid.go [hunk 2]
```diff
@@ -50,21 +50,23 @@ type ExtBidPrebidCacheBids struct {
...
 	DChain               json.RawMessage `json:"dchain,omitempty"`
+	DemandSource         string          `json:"demandSource,omitempty"`
 	MediaType            string          `json:"mediaType,omitempty"`
 	NetworkID            int             `json:"networkId,omitempty"`
 	NetworkName          string          `json:"networkName,omitempty"`
 	PrimaryCategoryID    string          `json:"primaryCatId,omitempty"`
+	RendererName         string          `json:"rendererName,omitempty"`
+	RendererVersion      string          `json:"rendererVersion,omitempty"`
 	SecondaryCategoryIDs []string        `json:"secondaryCatIds,omitempty"`
-	AdapterCode          string          `json:"adaptercode,omitempty"`
```
Comment (SyntaxNode) [inline openrtb_ext/bid.go:67]: Alphabetized the list and added `RendererName` + `RendererVersion`.

File: openrtb_ext/request.go [hunk 3]
```diff
@@ -41,52 +41,54 @@ type ExtRequest struct {
...
-
-	// Macros specifies list of custom macros along with the values. This is used while forming
-	// the tracker URLs, where PBS will replace the Custom Macro with its value with url-encoding
-	Macros            map[string]string               `json:"macros,omitempty"`
-	AdServerTargeting []AdServerTarget                `json:"adservertargeting,omitempty"`
-	BidAdjustments    *ExtRequestPrebidBidAdjustments `json:"bidadjustments,omitempty"`
-	// ReturnAllBidStatus if true populates bidresponse.ext.prebid.seatnonbid with all bids which was
-	// either rejected, nobid, input error
-	ReturnAllBidStatus bool `json:"returnallbidstatus,omitempty"`
 }
```
Comment (SyntaxNode) [inline openrtb_ext/request.go:92]: Alphabetized, organized the groupings, and added `sdk`.

**Review by AlexBVolcy — APPROVED** [review]
LGTM

**Review by VeronikaSolovei9 — APPROVED** [review]
Local testing looks good. Do we just want to pass values to adapters, correct? Will new meta fields RendererName  and RendererVersion be used in future?

**Comment by SyntaxNode** [comment]
> Do we just want to pass values to adapters, correct?

Correct. This information will be made available to adapters to consider in placing their bids. In the future, there might be a filtering module. 

> Will new meta fields RendererName and RendererVersion be used in future?

Bidders can use these fields immediately to share that information with the SDK.

**Pull request merged**